src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1210,13 +1210,13 @@ void TorrentHandle::setName(const QString &name)
     }
 }
 
-bool TorrentHandle::setCategory(const QString &category)
+bool TorrentHandle::setCategory(const QString &category, const QString &savePath)
```

Comment:
Please revert this change. It produces incorrect interface.

Reply:
The bigger issue is that this function already supports creating a new category. I'll refactor the existing code.

Reply:
If you want to change Core logic, please do it in separate PR. It isn't required here.

Reply:
>The bigger issue is that this function already supports creating a new category.

In a good way, we should allow only assign an existing category to the torrent.